vllm/attention/backends/abstract.py

```diff
@@ -195,6 +205,8 @@ def validate_configuration(
                 invalid_reasons.append("non-sparse not supported")
         if not cls.supports_compute_capability(device_capability):
             invalid_reasons.append("compute capability not supported")
+        if not cls.supports_attn_type(attn_type):
+            invalid_reasons.append(f"attention type {attn_type} not supported")
```

Comment:
```suggestion
            invalid_reasons.append(f"attention type not supported")
```
For consistency with the others.
I'd prefer to log the argument values in one place in `get_attn_backend_cls` to keep the log output shorter